src/settings/MappingProfiles/detailsSections/edit/OrderDetailSection/POLineDetails.js

```diff
@@ -112,27 +106,20 @@ export const POLineDetails = ({
 
   const onReceivingWorkflowParse = useCallback(
     value => {
-      setReceivingWorkflow(value);
-      if (value === `"${RECEIVING_WORKFLOW.INDEPENDENT}"`) return BOOLEAN_STRING_VALUES.TRUE;
-      if (value === `"${RECEIVING_WORKFLOW.SYNCHRONIZED}"`) return BOOLEAN_STRING_VALUES.FALSE;
+      if (value === `"${RECEIVING_WORKFLOW.INDEPENDENT}"`) return `"${BOOLEAN_STRING_VALUES.TRUE}"`;
```

Comment:
@mariia-aloshyna RECEIVING_WORKFLOW.INDEPENDEN is it not string? Not sure that we need wrap it with quotes.
`if (value === RECEIVING_WORKFLOW.INDEPENDENT) return BOOLEAN_STRING_VALUES.TRUE;`

The same questuin for all other places. Maybe we can create some kind of map here, for example:
```
const map = {
 [RECEIVING_WORKFLOW.INDEPENDENT]: BOOLEAN_STRING_VALUES.TRUE,
 [RECEIVING_WORKFLOW.SYNCHRONIZED]: BOOLEAN_STRING_VALUES.FALSE
// all other fields
}
```

we can avoid all of these IF statements.

Reply:
Thanks for clarification